src/CommandingOfficers/DocLight.java

```diff
@@ -0,0 +1,104 @@
+package CommandingOfficers;
+
+import Engine.GameScenario;
+import Engine.Combat.StrikeParams.BattleParams;
+import Engine.GameEvents.GameEventQueue;
+import UI.UIUtils;
+import Units.*;
+
+public class DocLight extends RuinedCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Doctor Light", UIUtils.MISC);
+      infoPages.add(new InfoPage(
+            "Even here, it is not safe. Even this grave has been defaced.\n"
+          + "Someone has written on this stone: 'Hope rides alone'\n"));
+      infoPages.add(new InfoPage(
+          "Base Zone: 0\n" +
+          "Can deploy your COU only twice, with different stats each time (Protoman is first)\n" +
```

Comment:
I read this as being able to deploy both in tandem at first (as is seemingly implied by the artwork) - but it appears that it actually means only twice ever, full stop. Would be worth emphasizing that it means only two CO deployments over an entire game, unlike other deployable COs (so choose wisely).

Reply:
I thought about having that be the mechanic, but it's too strong for what I'm going for, hah.
IMO that's a weird reading of what I wrote, since the simple way to express that is "Can deploy two COUs at once".

Reply:
Right, the issue is just that the implemented mechanic differs from a) what the art seems to indicate and b) what I therefore naturally expected. And I like to word things precisely and explicitly.

Reply:
"Cannot deploy a COU if your COU dies twice. Uses the Megaman statline after your COU dies.", then?

Reply:
I'd just add parens: "Can deploy your COU only twice (per game!), with..."